plugins/DileptonCategories.cc

```diff
@@ -0,0 +1,117 @@
+#include <cp3_llbb/Framework/interface/MuonsProducer.h>
+#include <cp3_llbb/Framework/interface/ElectronsProducer.h>
+#include <cp3_llbb/Framework/interface/JetsProducer.h>
+
+#include <cp3_llbb/Framework/interface/DileptonCategories.h>
+#include <cp3_llbb/Framework/interface/DileptonAnalyzer.h>
+
+// ***** ***** *****
+// Dilepton Mu-Mu category
+// ***** ***** *****
+bool MuMuCategory::event_in_category_pre_analyzers(const ProducersManager& producers) const {
+    const MuonsProducer& muons = producers.get<MuonsProducer>("muons");
+    return muons.p4.size() >= 2 ;
+};
+
+bool MuMuCategory::event_in_category_post_analyzers(const ProducersManager& producers, const AnalyzersManager& analyzers) const {
+    const DileptonAnalyzer& dilepton_analyzer = analyzers.get<DileptonAnalyzer>("dilepton_analyzer");
```

Comment:
The name of the ``dilepton_analyzer`` defined in the python configuration is ``dilepton``, not ``dilepton_analyzer``.